python/private/pypi/parse_requirements.bzl

```diff
@@ -92,7 +92,7 @@ def parse_requirements(
         # are returned as just the base package name. e.g., `foo[bar]` results
         # in an entry like `("foo", "foo[bar] == 1.0 ...")`.
         requirements_dict = {
-            normalize_name(entry[0]): entry
+            entry[1]: entry
```

Comment:
`parse_requirements` and `parse_requirements_txt` have unit tests in `//tests/pypi/` It would be better to add a test there instead of an example.